roles/pulp/defaults/main.yml

```diff
@@ -8,6 +8,9 @@ pulp_install_dir: '/usr/local/lib/pulp'
 pulp_install_plugins: {}
 pulp_install_api_service: true
 pulp_user: pulp
+pulp_user_id:
+pulp_group: pulp
+pulp_group_id:
```

Comment:
Please add these to the role README.

Also, what is the behavior when blank like this?

Reply:
The behavior is that the operating system assigns the UID. However, if it is provided, that number will be used. unless it's taken and tehn an error will occur.